### PR TITLE
Updated documentation to include several examples

### PR DESCRIPTION
**Pull request by lukasfolle**
Please feel free to suggest improvements!

### CODE REVIEW

File: tensorflow/python/keras/backend.py [hunk 1]
```diff
@@ -2918,6 +3001,19 @@ def stack(x, axis=0):
...
+
+  Example:
+      ```python
+        >>> a = tf.constant([[1, 2],[3, 4]])
+        >>> b = tf.constant([[10, 20],[30, 40]])
+        >>> tf.keras.backend.stack((a, b))
+        <tf.Tensor: id=146, shape=(2, 2, 2), dtype=int32, numpy=
+        array([[[ 1,  2],
+                [ 3,  4]],
+
```
Comment (tanzhenyu) [inline tensorflow/python/keras/backend.py]: what is the blank line here? is it supposed to be "..."?

Comment (lufol) [reply tensorflow/python/keras/backend.py]: The empty lines are returned by Python. I deleted them in 7d2750d to remove unnecessary lines.

**Comment by lufol** [comment]
@tanzhenyu Could you please review the changes?

**Review by tanzhenyu — DISMISSED** [review]

**Comment by gbaned** [comment]
@lufol Can you please check Ubuntu Sanity errors? Thanks!

**Review by lukasfolle — COMMENTED** [review]
Thanks for the review. @gbaned or @tanzhenyu can you approve the changes please?

**Comment by lufol** [comment]
Can some one please check the changes I have made?
@gbaned @tanzhenyu

**Comment by lufol** [comment]
@gbaned  Could you possibly assign a new reviewer as the last update is more than a week old?

File: tensorflow/python/keras/backend.py [hunk 2]
```diff
@@ -2492,6 +2492,17 @@ def concatenate(tensors, axis=-1):
 
   Returns:
       A tensor.
+
+  Example:
+      ```python
+      >>> a = tf.constant([[1, 2, 3], [4, 5, 6], [7, 8, 9]])
+      >>> b = tf.constant([[10, 20, 30], [40, 50, 60], [70, 80, 90]])
+      >>> tf.keras.backend.concatenate((a, b), axis=1)
```
Comment (tanzhenyu) [inline tensorflow/python/keras/backend.py]: say -1 even though it's the same in this case, since that's the default arg value.

**Comment by tanzhenyu** [comment]
Apology for the late reply. 2.0 has been kept us very busy.

File: tensorflow/python/keras/backend.py (hunk 2 above)
Comment (lufol) [reply tensorflow/python/keras/backend.py]: Thanks, 56d39f9 brings the change.

**Review by tanzhenyu — APPROVED** [review]
